### PR TITLE
Task/implement history log of parent blocking

### PR DESCRIPTION
**Pull request by SergeyGSA**
- added history log of parents blocking
- reworked filters of history log

### CODE REVIEW

File: src/app/shell/admin-tools/data/history-log/history-log-filters/history-log-filters.component.ts [hunk 1]
```diff
@@ -1,41 +1,93 @@
...
-      dateFrom: new FormControl(monthAgoDate),
-      dateTo: new FormControl(currentDate),
-      options: new FormControl('')
-    });
-
     this.filterData.emit(this.filtersForm.value);
   }
+  
+  public ngOnChanges(changes: SimpleChanges): void {
+    this.setBaseFiltersForm();
```
Comment (humanmantis) [inline src/app/shell/admin-tools/data/history-log/history-log-filters/history-log-filters.component.ts:29]: Do I need to do this every time a change occurs, or only when switching tabs?

File: src/app/shell/admin-tools/data/history-log/history-log.component.ts [hunk 2]
```diff
@@ -163,4 +174,26 @@ export class HistoryLogComponent implements OnInit, OnDestroy {
     Util.setFromPaginationParam(this.filters, this.currentPage, this.totalAmount);
     this.dispatchProperValue(this.tabIndex, this.filters, searchString);
   }
+
+  private initSubscribeOnEachHistory(): void {
```
Comment (humanmantis) [inline src/app/shell/admin-tools/data/history-log/history-log.component.ts:178]: It looks like we have multiple subscriptions to perform the same action.
Can we combine them using a merge operator or some other option?

**Review by humanmantis — CHANGES_REQUESTED** [review]

File: src/app/shell/admin-tools/data/history-log/history-log-filters/history-log-filters.component.ts (hunk 1 above)
Comment (SergeyGSA) [reply src/app/shell/admin-tools/data/history-log/history-log-filters/history-log-filters.component.ts:32]: We need this only when switching tabs, but I can't find better solution. Could you help me with this?

Comment (SergeyGSA) [reply src/app/shell/admin-tools/data/history-log/history-log-filters/history-log-filters.component.ts:29]: We need this only when tabs switching, so I can move this code to the component on higher level. There are mat-tabs in this component and I can observe tabs changing and execute this methods. What do you think?

Comment (humanmantis) [reply src/app/shell/admin-tools/data/history-log/history-log-filters/history-log-filters.component.ts:29]: We can use a setter for input property (tabName in my understanding) to track that the tab has been changed and reset the form only in this case.
E.g: 
```

  @Input public set tabName(newValue) {
    this._tabName = newValue;
    // reset form
    // ...
  }

  public get tabName() {
    return this._tabName;
  }

```

**Review by humanmantis — CHANGES_REQUESTED** [review]

**Review by humanmantis — APPROVED** [review]

**Pull request merged**